utils/build-validate-quickstart-artifact.ts

```diff
@@ -88,6 +89,24 @@ const main = () => {
   }
 
   console.log('[*] Validation succeeded');
+
+  if (shouldOutputArtifact) {
+    outputArtifact(artifact);
+  }
+}
+
+const outputArtifact = (artifact: Artifact) => {
+  console.log('[*] Outputting the artifact');
+  try {
+    fs.mkdirSync('./build', { recursive: true });
+
+    // Dump the artifact to artifact.json
+    fs.writeFileSync('./build/artifact.json', JSON.stringify(artifact));
+    // Copy the schema to schema.json
+    fs.copyFileSync('./schema/artifact.json', './build/schema.json');
+  } catch (e) {
+    console.error('Error writing artifact to file:', e);
+  }
```

Comment:
Would we want to error out if we c ant write to artifact file?  I'd imagine we'd want some indication of that so if the artifact build fails we can have some kind of notification

Reply:
Oh yeah, I suppose we could `exit(1)` here. The actual workflow will fail if there are no files to release so we're protected in that regard.